core/internal/gethwrappers/go_generate_test.go

```diff
@@ -52,52 +49,8 @@ func TestCheckContractHashesFromLastGoGenerate(t *testing.T) {
 		"should never differ!")
 }
 
-// TestArtifactCompilerVersionMatchesConfig compares the solidity version in the contract artifacts
```

Comment:
How do we ensure that the golang wrappers are built from ABI outputs compiled with the correct solidity version, without this test?

IIRC, this test is designed to ensure that if someone has old ABI outputs lying around and the solc minor version is increased in the config, they know to recompile with the expected solc version.

Reply:
We still have `TestCheckContractHashesFromLastGoGenerate` and `compareCurrentCompilerArtifactAgainstRecordsAndSoliditySources` tests which do I think what you're referring to.

The reason we're removing this is that we don't use the JSON artefacts compiled by belt anymore, we use raw `.abi` and `.bin` files. These files don't have any custom version fields to check against like the JSON files do.

Reply:
Thanks.